src/servers/sapphire_zone/Zone/House.cpp

```diff
@@ -122,6 +132,11 @@ uint32_t Core::House::getHouseId() const
   return m_houseId;
 }
 
+Core::Common::FFXIVARR_POSITION3 Core::House::getDoorPosition() const
```

Comment:
I just checked and these positions don't match all the doors, plot 3 goblet is an example of this where its inside the front wall to the right

Reply:
it looks like it's the world position of where the icon is on the map

Reply:
Which the exd name kindof gives away ;)